specs/src/ante_handler_v3.md

```diff
@@ -3,6 +3,7 @@
 The AnteHandler chains together several decorators to ensure the following criteria are met for app version 3:
 
 - The tx does not contain any messages that are unsupported by the current app version. See `MsgVersioningGateKeeper`.
+- The tx size is not larger than the application's configured versioned constant MaxTxBytes.
```

Comment:
[not blocking] we should have considered naming it something else to avoid conflicting with the mempool parameter `MaxTxBytes`. For example `MaxTxSize` would've worked

Reply:
I can change it in a follow-up since i need to update the ante handler anyway :)